docs/reference/continuous-export.md

```diff
@@ -0,0 +1,169 @@
+branches.
+
+### CLI
+
+The command `lakectl export` is the entrypoint to all export commands.
+
+#### `lakectl export get`
+
+Read the current export configuration for a branch using
+```shell
+lakectl export get lakefs://REPO@BRANCH
+```
+
+For instance,  `./lakectl export get lakefs://foo@master`.
+
+#### `lakectl export set`
+
+Set an export configuration for a branch by using
+```shell
+lakectl export set lakefs://REPO@BRANCH FLAGS...
+```
+
+Use these flags:
+* `--continuous`: enables _continuous export_, after every commit or merge to the branch.
+  Must be specified, so set `--continuous=false` to immediately start one single export.
+* `--path`: specifies the destination path to export to.
+* `--status-path`: specifies the export status path, under which a file is written giving the
+  status after each export.
+* `--prefix-regex`: identifies prefixes (directories) in which to place "export done" objects
+  (useful for triggering workflows).  May be repeated to specify multiple regular expressions.
+  
+#### `lakectl export run`
+
+Start exporting the requested branch now by using
+```shell
+lakectl export run lakefs://REPO@BRANCH
+```
+
+#### `lakectl export repair`
+
+If an export fails, for instance if the daemon has no permissions to write to the export
+directory, lakeFS has no way of determining how best to proceed or when to try to proceed.
+After repairing the issue, e.g. by granting permissions, restart by using
+```shell
+lakectl export repair lakefs://REPO@BRANCH
+```
+
+### API
+
+#### Configuration
+
+Configure and query continuous export configuration of a branch with a `PUT` or a `GET` to
+`/repositories/{repository}/branches/{branch}/continuous-export`.  Pass these JSON
+parameters in the body:
+```json
+{
+  "exportPath": "s3://company-bucket/path/to/export", // export objects to this path (required)
+  "exportStatusPath": "s3://company-bucket/path/to/status", // write status to this path (optional)
+  "lastKeysInPrefixRegexp": [
+    // list of regexps of keys to exported last in each prefix (for signalling) (optional)
+    ".*/_SUCCESS"
+  ]
+}
+```
+Once set on a branch, every commit or merge to that branch will be exported.
+
+#### Operation
+
+Export current branch state just once, without continuous operation, with `POST` to
+`/repositories/{repository}/branches/{branch}/export-hook`.
+
+Resume export after repairing with a `POST` to
+`/repositories/{repository}/branches/{branch}/repair-export`.
+
+## Operation
+
+When continuous export is enabled for a branch its current state is exported to S3.  From
+then on, every commit or merge operation to that branch will be exported.  The commit or
+merge operation on the branch returns as soon as export starts.  Multiple commits to the
+same exported branch may cause some otherwise-overlapping exports to be skipped.
+Eventually the last export will appear.  This is in line with "eventual consistency" of
+S3.
```

Comment:
That last line is no longer true.

Reply:
This one still is: when there are multiple updates to the same file, they show up in order until the last one does.  Updating to explain you can use status files to help.